### PR TITLE
briefing phase

### LINKED ISSUE #484: Pre game brief
When the host starts the game, there should be a short phase where everyone sees an encyclopedia article for the role they are about to play. 

It should check what role they actually are.
So if you get forger and instantly convert to godfather, it should show you the article for godfather.

I don't think there should be a ready up button. We can discuss this. 

It should go to evening after 15 seconds.

### LINKED ISSUE #500: No Trials Left Message
When there are no trials remaining, there should be a message sent in chat saying that there are no trials remaining.

### LINKED ISSUE #521: Rename phases
Morning -> Obituary
Voting -> Nomination
Evening -> Final Words

This should be done in the lang file
Potentially could be done in code but it would need a thorough review

### LINKED ISSUE #524: Error display overwrites game mode editor + other cover cards
When an error appears it closes the game mode editor

### PR DESCRIPTION
**Pull request by ItsSammyM**
closes #524
closes #521
closes #484
closes #500

adds dusk phase

### CODE REVIEW

File: client/src/menu/game/gameScreenContent/ChatMenu.tsx [hunk 1]
```diff
@@ -76,6 +81,10 @@ function ChatMessageSection(): ReactElement {
             }
         }
 
+        if (GAME_MANAGER.state.stateType === "game") {
+            setMessages(GAME_MANAGER.state.chatMessages)
+        }
+
```
Comment (Jack-Papel) [inline client/src/menu/game/gameScreenContent/ChatMenu.tsx:87]: What is this? Does it fix #469?

File: client/src/resources/keywords.json [hunk 2]
```diff
@@ -45,13 +45,13 @@
...
     },
-    "phase.morning": {
+    "phase.obituary": {
         "link": "standard/phases_and_timeline"
     },
     "phase.discussion": {
         "link": "standard/phases_and_timeline"
     },
-    "phase.voting": {
+    "phase.nomination": {
```
Comment (Jack-Papel) [inline client/src/resources/keywords.json:54]: Where's briefing?

Comment (Jack-Papel) [reply client/src/resources/keywords.json:54]: And dusk

File: server/tests/kit/game.rs [hunk 3]
```diff
@@ -28,7 +28,7 @@ impl TestGame {
     /// * When the specified day and phase is in the past.
     /// * If this would take the game to a day past the maximum day
     pub fn skip_to(&mut self, phase: PhaseType, day_number: u8) -> &PhaseState {
-        if day_number == 1 && phase < PhaseType::Evening {
+        if day_number == 1 && phase < PhaseType::FinalWords {
```
Comment (Jack-Papel) [inline server/tests/kit/game.rs:31]: fix

**Review by Jack-Papel — CHANGES_REQUESTED** [review]

File: client/src/menu/game/gameScreenContent/ChatMenu.tsx (hunk 1 above)
Comment (ItsSammyM) [reply client/src/menu/game/gameScreenContent/ChatMenu.tsx:87]: This resolves a different issue that we never made an issue for

When the chat menu is first loaded it doesn't always show the messages. Now it does

File: client/src/resources/keywords.json (hunk 2 above)
Comment (ItsSammyM) [reply client/src/resources/keywords.json:54]: i added them now

File: server/tests/kit/game.rs (hunk 3 above)
Comment (ItsSammyM) [reply server/tests/kit/game.rs]: all tests now pass

File: client/src/menu/game/gameScreenContent/ChatMenu.tsx (hunk 1 above)
Comment (Jack-Papel) [reply client/src/menu/game/gameScreenContent/ChatMenu.tsx:87]: Can you check if it solves #469 tho? It looks like it should

**Review by Jack-Papel — APPROVED** [review]

**Pull request merged**